### ISSUE TITLE
Command line argument to launch chatterino with a specified tab selected.

### ISSUE THREAD

**Issue by jp1995** [Chatterino/chatterino2#2498]
There has been some talk about this already, in #2484 and #2460, as well as streamlink/streamlink-twitch-gui#607.

**What should be added?**
I'm suggesting a command line argument, maybe something like `-a, --active <channel>`, that launches chatterino with that specified tab selected. If there is no such tab in the layout, create it.

**Why should it be added?**
Right now when chatterino is launched, the user likely has a number of tabs open. Whichever tab was active (selected) when the user closed chatterino will be active now. This means that the user will likely have to do manual labor by selecting the tab (or opening a new tab) that corresponds to the stream that was just launched alongside chatterino.

Basically it would be a useful addition for scripting and for using chatterino in conjunction with something like streamlink.

I realise that I'm asking for functionality that automates a 1 second mouse move and click, but it would be nice to have this one day.
